aea/cli/check_packages.py

```diff
@@ -603,16 +603,7 @@ def check_pypi_dependencies(configuration_file: Path) -> None:
 @click.option("--vendor", type=str, default=None, required=False)
 @pass_ctx
 def check_packages(ctx: Context, vendor: Optional[str]) -> None:
-    """
-    Run different checks on AEA packages.
-
-    Namely:
-    - Check that every package has existing dependencies
-    - Check that every package has non-empty description
-
-    :param ctx: AEA cli context.
-    :param vendor: filter by author name
-    """
+    """Run different checks on AEA packages."""
```

Comment:
This produces following command help string

```
Usage: autonomy check-packages [OPTIONS]

  Run different checks on AEA packages.

  Namely: - Check that every package has existing dependencies - Check that
  every package has non-empty description

  :param ctx: AEA cli context. :param vendor: filter by author name

Options:
  --vendor TEXT
  --help         Show this message and exit.
 ```
 
 This is a limitation of click